src/webportal/src/app/job/job-view/fabric/job-detail/components/summary.jsx

```diff
@@ -94,8 +94,10 @@ export default class Summary extends React.Component {
       modalTitle: '',
       autoReloadInterval: 10 * 1000,
       hideDialog: true,
+      currentAttemptID: this.props.jobInfo.jobStatus.retries,
```

Comment:
I will implement an independent job attempts page. Please seperate the frontend works and check in the api changes only.